pymodbus/client/mixin.py

```diff
@@ -39,7 +40,8 @@ class ModbusClientMixin:  # pylint: disable=too-many-public-methods
     def __init__(self):
         """Initialize."""
 
-    def execute(self, request: ModbusRequest) -> ModbusResponse:
+    @abstractmethod
```

Comment:
it returns a ModbusResponse when overwritten, so it should be defined as such. Remember this is an API spec for the library.

This is an. abstract method, but I strongly prefer not to define it as such, because it can be called from an app.

we have a lot of methods that are overwritten but that does not make them abstract. only those that must be overwritten should be declared as abstract.

The method can return a modbusResponse()